### PR TITLE
Add TFlite model to 002-openvino-api notebook

### PR DESCRIPTION
**Pull request by sbalandi**
+ fix rebase issue for 215 notebook

### CODE REVIEW

**Comment by sbalandi** [comment]
@eaidova I use model https://tfhub.dev/tensorflow/lite-model/inception_v4_quant/1/default/1 here. You recommended use [imagenet/mobilenet_v3_small_075_224/classification](https://tfhub.dev/google/lite-model/imagenet/mobilenet_v3_small_075_224/classification/5/default/1) ,  but I faced with fail about unsupported operation MEAN . Do you know, it is expected ?

**Comment by eaidova** [comment]
> @eaidova I use model https://tfhub.dev/tensorflow/lite-model/inception_v4_quant/1/default/1 here. You recommended use [imagenet/mobilenet_v3_small_075_224/classification](https://tfhub.dev/google/lite-model/imagenet/mobilenet_v3_small_075_224/classification/5/default/1) , but I faced with fail about unsupported operation MEAN . Do you know, it is expected ?

probably, yes. inceptionv4 is also fine

File: notebooks/002-openvino-api/002-openvino-api.ipynb [hunk 1]
```diff
@@ -240,6 +240,45 @@
...
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "8e7ba1b1",
+   "metadata": {},
+   "source": [
+    "### TensorFlow Lite Model\n",
+    "\n",
+    "[TFLite](https://www.tensorflow.org/lite) models saved for inference can also be passed to OpenVINO Runtime. Pass the filename with extension to `read_model` and exported an OpenVINO IR with `serialize`"
```
Comment (eaidova) [inline notebooks/002-openvino-api/002-openvino-api.ipynb:251]: which extension?

This sentence looks unfinished

File: notebooks/002-openvino-api/002-openvino-api.ipynb [hunk 2]
```diff
@@ -240,6 +240,45 @@
...
+   "id": "ed67c6c7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from openvino.runtime import Core\n",
+    "\n",
+    "ie = Core()\n",
+    "tflite_model_path = \"model/classification.tflite\"\n",
+    "\n",
+    "model_tflite = ie.read_model(model=tflite_model_path)\n",
```
Comment (eaidova) [inline notebooks/002-openvino-api/002-openvino-api.ipynb:266]:
```suggestion
    "model_tflite = ie.read_model(flite_model_path)\n",
```

File: notebooks/002-openvino-api/002-openvino-api.ipynb (hunk 1 above)
Comment (sbalandi) [reply notebooks/002-openvino-api/002-openvino-api.ipynb]: add `.tflite`

File: notebooks/002-openvino-api/002-openvino-api.ipynb (hunk 2 above)
Comment (sbalandi) [reply notebooks/002-openvino-api/002-openvino-api.ipynb]: removed

**Review by eaidova — APPROVED** [review]

**Pull request merged**